lib/xapi/docs.rb

```diff
@@ -0,0 +1,49 @@
+module Xapi
+  # Some docs
+  class Docs
+    AUTHORIZED_DOCS = [
+      "HELLO.md",
+      "WORKFLOW.md",
+      "RESOURCES.md",
+      "INSTALLATION.md",
+      "TOOLS.md",
+    ]
```

Comment:
I was thinking for the json array, but it might make sense as both (using upcase to create the file names I guess)